querybook/webapp/components/DataDocQueryCell/DataDocQueryCell.scss

```diff
@@ -9,7 +9,12 @@
         flex-direction: column;
 
         .query-metadata {
-            padding: 0px 10px;
+            padding: 0px 8px;
+            .QueryRunButton,
+            > .Dropdown {
```

Comment:
can we reference the additional buttons dropdown with custom classname instead of `> .Dropdown`?